Standards/scs-0111-v1-volume-type-decisions.md

```diff
@@ -17,7 +17,7 @@ We want to standardize a few varieties of volume types. While a user can choose
 
 In General: what the different volume types are capable of is highly dependend on both the used backend and the configurations of OpenStack. A few options are worth being at least recommended.
 
-## Design Considerations
+## Context
```

Comment:
Though this is indeed a DR, I have the distinct feeling that the change is wrong. The section does indeed cover several options that have been considered, and it is not about context so far as I can see.

Reply:
See https://github.com/SovereignCloudStack/standards/pull/441#discussion_r1549764594